src/Session/CacheSessionPool.php

```diff
@@ -1,1048 +0,0 @@
-<?php
```

Comment:
Cloud you also remove excludePaths setting in phpstan.neon?

https://github.com/colopl/laravel-spanner/blob/master/phpstan.neon#L6-L8

Reply:
It has been removed. Thanks.